.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
+max-line-length = 120
+import-order-style = pycharm
+enable-extensions = G
+ignore =
+    W504, # line break occurred after a binary operator
+    # allow only generator_stop and annotations future imports
+    FI10,FI11,FI12,FI13,FI14,FI15,FI16,FI17,FI18,FI55,FI58,
+    C814, # missing trailing comma in Python 2 only
+    C818  # trailing comma on bare tuple prohibited
```

Comment:
Do we need to block this?

Reply:
I just (obviously) copied this from the judge repo. It's probably best to keep a constant style across all our projects.

Reply:
The only reason we have is because in the judge, `C818` requires

```python
    @classmethod
    def get_versionable_commands(cls):
        return ((cls.command, cls.get_command()),)
```

whereas PyCharm wants

```python
    @classmethod
    def get_versionable_commands(cls):
        return (cls.command, cls.get_command()),
```

We should probably make a ruling on this.

Reply:
The first is strictly harder to misread, so I'm in favour of it.

Reply:
Then ignore `C818` and I'll drop it from judge.